services/nomad/build/build-rsyncd.nomad

```diff
@@ -0,0 +1,58 @@
+job "build-rsyncd" {
+  type = "service"
+  datacenters = ["VOID"]
+  namespace = "build"
+
+  group "rsync" {
+    count = 1
+    network {
+      mode = "bridge"
+      port "rsync" { to = 873 }
+    }
+
+    volume "root-pkgs" {
+      type = "host"
+      source = "root-pkgs"
+      read_only = false
+    }
+
+    service {
+      name = "build-rsyncd"
+      port = "rsync"
+    }
+
+    task "rsyncd" {
+      driver = "docker"
+
+      config {
+        image = "ghcr.io/void-linux/infra-rsync:20230812"
+        volumes = [ "local/buildsync.conf:/etc/rsyncd.conf.d/buildsync.conf" ]
```

Comment:
Same issue as was mentioned in lsyncd, this will shadow if the underlying template changes.  I don't think this matters since this one appears to be static though.

Reply:
yeah it's static and I don't intend to template it. i guess it could be `data = file("buildsync.conf")`